docs/get-started.rst

```diff
@@ -102,6 +102,104 @@ with a *PostgreSQL* container.
 Using the Python package
 ========================
 
+Installing Python 2.7/3.4+ on Ubuntu
+------------------------------------
+
+Use the following commands to install prerequisites for Python:
+
+::
+
+    sudo apt-get install build-essential checkinstall
+    sudo apt-get install libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev
+
+Download Python 2.7 using the following commands. You can also download the latest version in place of the one specified below.
+
+::
+
+    cd ~/Downloads/
+    wget http://wwww.python.org/ftp/python/2.7.5/Python-2.7.5.tgz
```

Comment:
why are you installing python this way rather than relying on the system packaged version?